File: C/linear search.c
#include<stdio.h>

int search (int arr[],int n,int x){
int i;
for(i=0;i<n;i++){
    if(arr[i]==x)
    return i;
}
    return -1;

}
int main(void){

int arr[]={2, 3, 4, 5 ,78, 68, 45 ,99};
int x=45;
int n=sizeof(arr)/sizeof(arr[0]);
for(int i=0;i<n;i++){
    printf("%d ",arr[i]);
}
printf("\n");
printf("Enter the element you want to search in arr[]: ");
scanf("%d",&x);


int result=search(arr,n,x);
(result==-1)?printf("Element is not present in array ")
            :printf("Element is present at index %d",result);

             return 0;
}

File: Java/linearSearch.java
import java.util.Scanner;

public class linearSearch {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("enter the size of the array: ");
        int n = sc.nextInt();
        int []marks = new int[n];

        //input loop 
        for(int i=0 ; i<n;i++){
            marks[i]=sc.nextInt();
        }
        System.out.println("Enter the number you want to search: ");
        int number = sc.nextInt();
        //output loop
        for(int j =0;j<n;j++){
            if(marks[j]== number){
                System.out.println("your desired number is at: "+ j + "th place");
            }
        }
    }

}
